modules/specifying-role-arn-albo-sts.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * networking/installing-albo-sts-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-role-arn-albo-sts_{context}"]
+= Specifying the role ARN for ALBO on an STS cluster
+
+The role ARN for the Operator needs to be passed to the Operator as an environment variable. You can use the dedicated input box in the OperatorHub web UI or specify it in the Subscription resource when installing the Operator by using the OpenShift CLI.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
```

Comment:
This is not necessary anymore as we suggest to use `new-project` and not `create namespace`.